client-spark/spark2/src/main/java/org/apache/spark/shuffle/RssShuffleManager.java

```diff
@@ -338,9 +337,15 @@ public <K, C> ShuffleReader<K, C> getReader(ShuffleHandle handle,
           + blockIdBitmap.getLongCardinality() + " blockIds for shuffleId[" + shuffleId + "], partitionId["
           + startPartition + "]");
 
+      final RemoteStorageInfo shuffleRemoteStorageInfo = rssShuffleHandle.getRemoteStorage();
```

Comment:
Could we extract a function in module `spark common`

Reply:
ok